src/test/shared/defaultAwsContext.test.ts

```diff
@@ -153,6 +163,56 @@ describe('DefaultAwsContext', function () {
         })
     })
 
+    it('sets default region to last region from prompter', async function () {
+        const regions = [
+            { id: 'us-west-2', name: 'PDX' },
+            { id: 'us-east-1', name: 'IAD' },
+            { id: 'foo-bar-1', name: 'FOO' },
+        ]
+        const p = createRegionPrompter(regions, {
+            title: 'Select regionnnn',
+            buttons: createCommonButtons('https://aws.amazon.com/'),
+            defaultRegion: 'foo-bar-1',
+        })
+        const tester = createQuickPickTester(p)
+        const selection = regions[0]
+        tester.assertItems(['FOO', 'PDX', 'IAD'])
+        tester.acceptItem({
+            label: selection.name,
+            detail: selection.id,
+            data: selection,
+            skipEstimate: true,
+            description: '',
+        })
```

Comment:
So in that case would we want to pass `selection.name` instead of `selection.id` since thats the label attribute? Or does it match based on `detail` not `label`?

Reply:
Oh yeah it'd be the region name, my bad.